### PR TITLE
Document cleanup

### PR DESCRIPTION
**Pull request by shilei-wang**
document fix

### CODE REVIEW

File: docs/zh/features/service.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ iris q service definition <service-name>
 
 ```json
 {
-  "price": "0.1iris",
+  "price": "0.1uiris",
```
Comment (chengwenxi) [inline docs/zh/features/service.md:81]: 0.1iris

File: docs/features/service.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ The pricing object specifies how the provider charges for its service; it must c
 
 ```json
 {
-  "price": "0.5iris",
+  "price": "0.5uiris",
```
Comment (chengwenxi) [inline docs/features/service.md:77]: 0.1iris

File: docs/cli-client/nft.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+```bash
+iris tx nft mint [denomID] [tokenID] [flags]
+```
+
+**Flags:**
+
+| Name, shorthand      | Required | Default | Description     |
+| ----------- | ----- | -------------------------------------- | ---- |
+| --uri       |  |  URI of off-line token data              |      |
```
Comment (secret2830) [inline docs/cli-client/nft.md:49]: off-line -> off-chain ?

File: docs/cli-client/nft.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+```bash
+iris query nft supply [denomID] 
+iris query nft supply [denomID] --owner=<owner address>
+```
+
+### iris query nft owner 
+
+```bash
+iris query nft owner [owner address]
+iris query nft owner --denom=<denomID> [owner address]
```
Comment (SegueII) [inline docs/cli-client/nft.md:110]: iris query nft owner [owner address] --denom=\<denomID\>

**Review by SegueII — APPROVED** [review]

**Review by chengwenxi — APPROVED** [review]

**Pull request merged**